### DISCUSSION TITLE
Which format to use for triple file when train?

### DISCUSSION THREAD

**Discussion by tonylmai** [stanford-futuredata/ColBERT#136] [Q&A] [answered]
The instruction for training ColBERT says that it requires a JSONL triples file with a [qid, pid+, pid-] list per line, for example triples="/path/to/MSMARCO/triples.train.small.tsv". But triples.train.small.tsv contains question, answer and passage. 

Which format should I use?

**Comment by hhhhzs666** [comment]
What is the training data format? The introduction is not clear

**Comment by okhat** [comment, marked as answer]
Both formats work. I suggest sticking to a jsonl format
